workflow-templates/npm-publish.yml

```diff
@@ -0,0 +1,31 @@
+name: Release CI
+on:
+  push:
+    branches:
+      - master
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
```

Comment:
Wouldn't we want to specify 20.04 like we do for the PyPI workflow, instead of potentially getting 20.10 or such before we officially support it?